core/bootstrap.py

```diff
@@ -113,21 +113,21 @@ def get_sample_unstable_addrs(self, num_addrs):
             i_warn_you_msg = "You are messing with my off-line detector, my friend"
             logger.warning(i_warn_you_msg)
         if 1:#len(self._unstable_ip_port) < num_addrs:
-            print '>>>len(self._unstable_ip_port) == %d, num_addrs: %d' % (
-                len(self._unstable_ip_port), num_addrs)
-            
+            logger.debug('>>>len(self._unstable_ip_port) == %d, num_addrs: %d' % (
+                len(self._unstable_ip_port), num_addrs))
```

Comment:
@rauljim These are just numbers, they should be fine. I will review and fix the others.